src/attributes.js

```diff
@@ -185,23 +185,25 @@ class Attributes {
         return new Proxy(attributes, {
             get (target, name, receiver) {
                 if (typeof name === 'string' && !Reflect.has(target, name)) {
-                    if (name.indexOf('get') !== -1) {
-                        // Undo Twig's property name to method name conversion.
-                        name = name.replace('get', '').toLowerCase();
-                        // Re-route into storage unless class property is requested.
-                        if (name === 'class') {
-                            name = 'classes';
-                        }
-                        else {
-                            target = target.storage;
-                        }
+                    // Undo Twig's property name to method name conversion.
+                    name = name.replace('get', '').toLowerCase();
```

Comment:
Doesn't this remove the substring "get" anywhere / wherever it may appear in the name instead of just the beginning?

Reply:
Please see the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Parameters) when a substring is given.
> Only the first occurrence will be replaced.

If you want to replace all occurrences you would have to pass a regex with the `/g` flag

Reply:
Oh, sorry, that's not what I meant. It looks like the getter is invoked for more than only `get*` methods now as you removed the condition `name.indexOf('get') !== -1`?

What if the attribute name that is being accessed is "tar**get**", as in `attributes.target`? Will that cause invocations of `hasTarget` as well as `getTarget`?

```suggestion
                    name = name.replace(/^get/, '').toLowerCase();
```